x-pack/filebeat/cmd/agent.go

```diff
@@ -15,8 +15,7 @@ import (
 )
 
 func filebeatCfg(rawIn *proto.UnitExpectedConfig, agentInfo *client.AgentInfo) ([]*reload.ConfigWithMeta, error) {
-	procs := defaultProcessors()
```

Comment:
Probably worth moving the defaultProcessors function into the file that uses it, since it isn't actually used here anymore.

This comment applies to each of the beats.

Reply:
Yah, good point, was kinda unsure of how to clean those up.